### PR TITLE
Update troubleshooting: On Gateway Timeout

### PR DESCRIPTION
**Pull request by jonathanlukas**
## Description

Sometimes, customers have a problem that a gateway timeout occurs. This has often been related to the `proxy-buffer-size` of the ingress (controller).

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by akeller** [comment]
@camunda/distribution, I see Jesse is OOO. Can someone else have a look?

**Review by christinaausley — APPROVED** [review]
Technical review in latest commit, though would wait until you have an engineering review to merge 👍

**Pull request merged**
